docs/docs/lavague-qa/examples.md

```diff
@@ -0,0 +1,489 @@
+# LaVague QA Examples
+
+!!! warning "Beta release"
+    LaVague QA is still a work in progress and may contain bugs. Join our [community of amazing contributors](https://discord.gg/invite/SDxn9KpqX9) to help make this tool more reliable!
+
+We will periodically add [examples in our repository](https://github.com/lavague-ai/LaVague/blob/main/lavague-qa/features/). In this section, we will go over each of the Gherkin files and showcase the generated code. 
+
+
+| URL                              | Link to file                                                  | Purpose                                                  |
+|:---------------------------------|:--------------------------------------------------------------|:---------------------------------------------------------|
+| https://amazon.fr/               | [demo_amazon.feature](https://github.com/lavague-ai/LaVague/blob/main/lavague-qa/features/demo_amazon.feature)       | Tests the Amazon search, cart and cart deletion to ensure expected behavior |
+| https://laposte.fr/              | [demo_laposte.feature](https://github.com/lavague-ai/LaVague/blob/main/lavague-qa/features/demo_laposte.feature)     | Tests an interactive shipping cost calculator to ensure expected output |
+| https://en.wikipedia.org/        | [demo_wikipedia.feature](https://github.com/lavague-ai/LaVague/blob/main/lavague-qa/features/demo_wikipedia.feature) | Tests the login feature of Wikipedia  |
+| https://hsbc.fr/                 | [demo_hsbc.feature](https://github.com/lavague-ai/LaVague/blob/main/lavague-qa/features/demo_hsbc.feature)           | Tests proper multi-tab navigation and cookie banners |
+
+
+In each of these examples, we'll showcase how LaVague QA can be used to generate tests for critical parts of your websites. 
```

Comment:
We don't really show how we used LaVague to do this. We just show the feature file and generated file. 

I would do one of the following or similar:
- Add some text about how to run the examples
- Add the command for running the examples with each example so it reads more as a guide
- Add end-to-end notebook examples with the installation, API key and then run and show files using 'cat' so users can really test it live.